### ISSUE TITLE
Hardened blocks

### ISSUE THREAD

**Issue by 3Lord3** [MLG-Fortress/ExtraHardMode#258]
This parameter is responsible for breaking the specified block with an iron or diamond pickaxe. But this is not written in the config. We need to clarify this, and preferably allow to specify tools for this function.

Hardened Blocks:
- STONE

Netherite pickaxe, by the way, does not count. Specified blocks can be broken with iron and diamond pickaxes, but not with netherite pickaxe.

**Comment by RoboMWM** [comment]
Pretty sure you can add whatever tool you want to allow to break stone in the config. Netherrite pickaxe is a default that will be added in an upcoming release if I haven't already pushed out a release.

**Comment by 3Lord3** [comment]
> Pretty sure you can add whatever tool you want to allow to break stone in the config.
Can you tell me which parameter is responsible for this?

> Netherrite pickaxe is a default that will be added in an upcoming release if I haven't already pushed out a release.
Okay

**Comment by RoboMWM** [comment]
pretty sure you pasted it in the other issue. Where you specify how many breaks of stone a tool can sustain.

**Comment by 3Lord3** [comment]
It specified how many blocks a specified pick had to break in order to break. But it seems you can't specify a tool for "Hardened Blocks:". All of the blocks specified there must be broken with an iron pick, and I don't see any way to change that.

**Comment by RoboMWM** [comment]
no that only applies to how many hardened stone blocks. Doesn't modify when used on any other block afaik.

**Comment by 3Lord3** [comment]
Yes, this indicates how many blocks of stone can be destroyed by the specified tool.But the setting that forces players to search open caves to soften stone for mining uses only iron and diamond picks (You'll add a netherite pick to this list in the next release). 
Can you specify in the comment that this only affects these picks, or allow you to specify which intruments are needed to break the "hardened block" ?

**Comment by RoboMWM** [comment]
I thought I posted a comment here, but I guess not... or maybe I posted it in the other one.

The tools you specify there are for breaking hardened stone. If a tool isn't specified there then it can't break it. So you can add netherite pickaxe there and then the netherite pickaxe can break it. Same with wooden or gold pickaxes or other tools, if you want to add other tools/items that can be held that will allow stone to be broken.

**Comment by 3Lord3** [comment]
You just described what is highlighted in red. But how do I adjust the tool for what is highlighted in black?
Or, are they interrelated? Then you should call the red setting "Amount of Hardened Blocks Tool Can Mine:"
![image](https://user-images.githubusercontent.com/62335161/102708698-d1244780-42bd-11eb-8ef1-8f480d35b2c6.png)

**Comment by RoboMWM** [comment]
I'm pretty sure BigScary intended it to be Stone, and Diemex later added options for other blocks to be "hardened."

If you want to PR a change (and migration) feel free... but ya, they are related yes.
